penn_chime/models.py

```diff
@@ -1,9 +1,118 @@
+from collections import namedtuple
 from typing import Generator, Tuple
 
 import numpy as np
 import pandas as pd
 import streamlit as st
 
+RateLos = namedtuple('RateLos', ('rate', 'length_of_stay'))
+
+
+class Parameters:
+
+    def __init__(
+        self, *,
+        current_hospitalized: int,
+        doubling_time: float,
+        known_infected: int,
+        market_share: float,
+        relative_contact_rate: float,
+        susceptible: int,
+
+        hospitalized: RateLos,
+        icu: RateLos,
+        ventilated: RateLos,
+    ):
+        self.current_hospitalized = current_hospitalized
+        self.doubling_time = doubling_time
+        self.known_infected = known_infected
+        self.market_share = market_share
+        self.relative_contact_rate = relative_contact_rate
+        self.susceptible = susceptible
+        self._n_days = 0
+
+        self.hospitalized = hospitalized
+        self.icu = icu
+        self.ventilated = ventilated
+
+        self.rates = tuple(
+            each.rate
+            for each in (hospitalized, icu, ventilated)
+        )
+        self.lengths_of_stay = tuple(
+            each.length_of_stay
+            for each in (hospitalized, icu, ventilated)
+        )
+
+        # TODO: I tried casting this to an int but this breaks unittest comparisons (third digit)
```

Comment:
IIUC sounds like you could fix the test? those were added with the thinking "something is better than nothing" - not meant to be The Source Of Truth

Reply:
i second sam's comment

Reply:
That comment was from PJ's merge.

Reply:
Actually, infected shouldn't be cast to an int here, it is a computed guess based off of the other numbers we do have.